lib/appsignal/transaction.rb

```diff
@@ -541,8 +537,54 @@ def to_h
     end
     alias_method :to_hash, :to_h
 
+    protected
+
+    attr_writer :ext, :is_duplicate
```

Comment:
```suggestion
    attr_writer :is_duplicate
```
I don't see where this is used. Can we remove it?